Userland/Libraries/LibGfx/ImageFormats/JPEGXLChannels.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2025, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Error.h>
+#include <AK/Vector.h>
+
+namespace Gfx {
+
+class Channel {
```

Comment:
The class is called "Channel" but the file "Channels". Will there be more channel classes in the future?

Reply:
Not that I planned, it's just an overlook on my side :disappointed: